src/NuGet.config

```diff
@@ -10,4 +10,9 @@
   <config>
     <add key="repositoryPath" value="..\packages" />
   </config>
+  <packageRestore>
+    <!-- Automated package restore in VS does not work at this time with
+         this project and it causes build failures in VS. Disable it. -->
```

Comment:
Just so I understand, this is essentially unchecking the `Automatically check for missing packages during build in Visual Studio` option in VS, but only for things in corefx?